templates/zerver/integrations/openshift.md

```diff
@@ -0,0 +1,54 @@
+Ths integration sends a notification every time a deployment is made in an OpenShift instance.
+
+First, download our [Pythonbindings and example scripts](/api), and move it to your local copy of the application’s
```

Comment:
A little error here, it should be `[Python bindings`. Another reason to use a macro for this. :)